_15thnight/forms.py

```diff
@@ -1,17 +1,33 @@
 from flask_wtf import Form
-from wtforms import TextField, PasswordField, BooleanField, SubmitField, TextAreaField, \
-    SelectField, IntegerField
-from wtforms.validators import DataRequired, Length, Email, EqualTo, Required
+from wtforms import (
+    TextField, PasswordField, TextAreaField, SelectField, SelectMultipleField,
+    IntegerField
+)
+from wtforms.validators import (
+    DataRequired, Email, EqualTo, Length, NumberRange
+)
 
+from _15thnight.models import Category
 
 
-USER_ROLES = [('provider', 'PROVIDER'), ('advocate', 'ADVOCATE'), ('admin', 'ADMIN')]
+USER_ROLES = [
+    ('provider', 'PROVIDER'), ('advocate', 'ADVOCATE'), ('admin', 'ADMIN')]
+
+# Needed for migrations, before the Category table is created
+try:
+    categories = [
+        (category.id, category.name) for category in Category.query.all()]
+except:
+    categories = []
```

Comment:
Yep. Category.query.all() was being called before the migrations had created the Category table and choked.